x-pack/platform/plugins/shared/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable_factory.tsx

```diff
@@ -179,25 +180,29 @@ export const getAnomalySwimLaneEmbeddableFactory = (
             defaultMessage: 'swim lane',
           }),
         onEdit: async () => {
-          try {
-            const { resolveAnomalySwimlaneUserInput } = await import(
-              './anomaly_swimlane_setup_flyout'
-            );
-
-            const result = await resolveAnomalySwimlaneUserInput(
-              { ...coreStartServices, ...pluginsStartServices },
-              parentApi,
-              uuid,
-              {
-                ...titleManager.getLatestState(),
-                ...swimlaneManager.getLatestState(),
-              }
-            );
-
-            swimlaneManager.api.updateUserInput(result);
-          } catch (e) {
-            return Promise.reject();
-          }
+          openLazyFlyout({
+            core: coreStartServices,
+            parentApi,
+            flyoutProps: {
+              focusedPanelId: uuid,
+            },
+            loadContent: async ({ closeFlyout }) => {
+              const { ResolveAnomalySwimlaneUserInput } = await import(
```

Comment:
same as in the previous PR, it's probably worth to move this import up at the top of the file because otherwise we create a chunk with heavily duplicated code from another async chunk.